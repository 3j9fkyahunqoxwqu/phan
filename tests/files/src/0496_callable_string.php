<?php
call_user_func(function() {
    $x = 'strlen';
    echo $x(rand(0,10), false);
});
