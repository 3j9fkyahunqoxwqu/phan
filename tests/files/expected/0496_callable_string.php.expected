%s:4 PhanParamTooManyInternal Call with 2 arg(s) to \strlen() which only takes 1 arg(s)
%s:4 PhanTypeMismatchArgumentInternal Argument 1 (string) is int but \strlen() takes string
